src/features/hotkeys-manager/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { TypedEventEmitterWithContext } from '~/shared/lib/typed-event-emitter';
+
+export type HotkeysManagerEvents = {
+  [key: string]: (alias: string, context: string, e: KeyboardEvent) => void;
```

Comment:
Try:

```TS
[K in `activated:${string}`]: (alias: string, context: string, e: KeyboardEvent) => void;
[K in `deactivated:${string}`]: (alias: string, context: string, e: KeyboardEvent) => void;
```

Also it would be nice to have general events: activated, deactivated. Which are triggered by any hotkey.

Reply:
Updated events type in a little bit different way. Thx
General events would be added if necessary